### DISCUSSION TITLE
Subdeck not working

### DISCUSSION THREAD

**Discussion by chenxin-yan** [st3v3nmw/obsidian-spaced-repetition#365] [Q&A]
Whether I use the hierarchy tags or sub folder method of creating subdecks, they do not work for me. Im not sure if this is a bug or i did something wrong.

**Comment by sameerad2001** [comment]
This plugin supports nested tags (subdecks)
Format : `#flashcards/topic/subTopic`

![image](https://user-images.githubusercontent.com/66942584/151109203-f96a7508-7c4c-4c6a-976d-4808116e9f2e.png)

hope this helps
